CHANGELOG.md

```diff
@@ -28,7 +28,8 @@
   ensuring that the parser correctly handles namespaced folders.  
   [David Jennes](https://github.com/djbe)
   [#199](https://github.com/AliSoftware/SwiftGen/pull/199)
-* Swift 3 migration [ahtierney](https://github.com/ahtierney)
+* Swift 3 migration
```

Comment:
Missing the full-stop + 2-spaces at the end of this line so that the rendered markdown is formatted properly